### DISCUSSION TITLE
Transmission daemon 3.0 Scrape error: Could not connect to tracker Ubuntu server 22.10

### DISCUSSION THREAD

**Discussion by williamthrilliam** [transmission/transmission#4293] [Support]
I can't for the life of me figure out what's wrong. Transmission-daemon will use upnp successfully and note that the port is open. But every single one of my torrents won't seed or download, save for one. [Ubuntu 22.10 server torrent](https://releases.ubuntu.com/22.10/ubuntu-22.10-live-server-amd64.iso.torrent?_ga=2.230656159.264445782.1669910269-2113982142.1668964206) downloaded 167MB then stalled out. Am I running into the [curl ssl issue](https://github.com/transmission/transmission/commit/44411d719c0bd6c17a794ab03eb8148f20a79b81#diff-a9ffa3cdcb7f1da80769c797a5af2c83)? 

I'm not sure I'm adding my environment variable to systemd correctly, can someone verify? The contents of /lib/systemd/system/transmission-daemon.service
`[Unit]
Description=Transmission BitTorrent Daemon
After=network.target

[Service]
User=debian-transmission
Environment=TR_CURL_SSL_NO_VERIFY=1
Type=notify
ExecStart=/usr/bin/transmission-daemon -f --log-info --logfile /var/log/transmission.log
ExecStop=/bin/kill -s STOP $MAINPID
ExecReload=/bin/kill -s HUP $MAINPID
NoNewPrivileges=true

[Install]
WantedBy=multi-user.target`

Any other ideas would be helpful. Thanks!

**Comment by ckerr** [comment]
FWIW I'm able to scrape that torrent with 4.0.0-beta.2:

```
$ transmission-show --version
transmission-show 4.0.0-beta.2 (bceb368f1b)
$ transmission-show --scrape ./ubuntu-22.10-live-server-amd64.iso.torrent\?_ga=2.230656159.264445782.1669910269-2113982142.1668964206
Name: ubuntu-22.10-live-server-amd64.iso
File: ./ubuntu-22.10-live-server-amd64.iso.torrent?_ga=2.230656159.264445782.1669910269-2113982142.1668964206

https://torrent.ubuntu.com/scrape?info_hash=_H%F6%8D%0E%7C%FE%2CL%CE%3D3%10O%CF%FA%F6%14Q5 ... 522 seeders, 63 leechers
https://ipv6.torrent.ubuntu.com/scrape?info_hash=_H%F6%8D%0E%7C%FE%2CL%CE%3D3%10O%CF%FA%F6%14Q5 ... 522 seeders, 63 leechers
```

**Reply by williamthrilliam** [reply]
@ckerr any tips on building 4.0 from source for transmission-daemon and web?

**Reply by ckerr** [reply]
https://github.com/transmission/transmission/blob/main/docs/Building-Transmission.md
